packages/atomic/src/components/search/facets/facet-segmented-value/facet-segmented-value.ts

```diff
@@ -0,0 +1,70 @@
+import {html} from 'lit';
+import {ifDefined} from 'lit/directives/if-defined.js';
+import {keyed} from 'lit/directives/keyed.js';
+import {renderButton} from '@/src/components/common/button';
+import type {FacetValuePropsBase} from '@/src/components/common/facets/facet-common';
+import type {FunctionalComponent} from '@/src/utils/functional-component-utils';
+
+export const renderFacetSegmentedValue: FunctionalComponent<
+  FacetValuePropsBase
```

Comment:
Just a note:

I noticed that we're not using the `class`, `part`, `additionalPart`, and `searchQuery` props at all in this functional component, so the FacetValuePropsBase is a bit broad for what we need.

However, this is consistent with the original Stencil implementation, and it matches the facet-value-checkbox (which also doesn't use these props).

We could define a narrower type for this component (or use Pick), but TBH I don't think it's worth the effort / added complexity.

Let's keep it as is, there's no harm really.